xbmc/cores/RetroPlayer/rendering/VideoRenderers/RPRendererOpenGL.cpp

```diff
@@ -306,20 +303,25 @@ void CRPRendererOpenGL::Render(uint8_t alpha)
   glEnableVertexAttribArray(vertLoc);
   glEnableVertexAttribArray(loc);
 
-  glGenBuffers(1, &indexVBO);
-  glBindBuffer(GL_ELEMENT_ARRAY_BUFFER, indexVBO);
+  glBindBuffer(GL_ELEMENT_ARRAY_BUFFER, m_mainIndexVBO);
   glBufferData(GL_ELEMENT_ARRAY_BUFFER, sizeof(GLubyte)*4, idx, GL_STATIC_DRAW);
 
   glUniform4f(uniColLoc,(colour[0] / 255.0f), (colour[1] / 255.0f), (colour[2] / 255.0f), (colour[3] / 255.0f));
+
   glDrawElements(GL_TRIANGLE_STRIP, 4, GL_UNSIGNED_BYTE, 0);
 
   glDisableVertexAttribArray(vertLoc);
   glDisableVertexAttribArray(loc);
 
   glBindBuffer(GL_ARRAY_BUFFER, 0);
-  glDeleteBuffers(1, &vertexVBO);
   glBindBuffer(GL_ELEMENT_ARRAY_BUFFER, 0);
-  glDeleteBuffers(1, &indexVBO);
 
   m_context.DisableGUIShader();
 }
+
+void CRPRendererOpenGL::Deinitialize()
```

Comment:
It was a big red flag that this function overrided but didn't call superclass Deinitialize(). I see that parent implementation is empty, but you shouldn't assume it will stay that way.